x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/enrich_signal_threat_matches.test.ts

```diff
@@ -94,7 +94,7 @@ describe('buildMatchedIndicator', () => {
     const indicators = buildMatchedIndicator({
       queries: [],
       threats,
-      indicatorPath: DEFAULT_INDICATOR_PATH,
+      indicatorPath: 'threat.indicator',
```

Comment:
if the intention is to keep this explicit, no need for modification - although it looks like we could declare an `indicatorPath` variable and assign it in the`beforeEach` block with the value `threat.indicator` or `INDICATOR_DESTINATION_PATH` so that we can use shorthand. In the same vein, it looks like there are a couple of tests that build the matched indicators in the same manner, so maybe it could make sense to declare a new var called `defaultIndicators` in the same `beforeEach` block where the `defaultIndicators` would be `buildMatchedIndicator({queries, threats, indicatorPath})`

Reply:
Agreed on DRYing this up, I think I'll do that here 👍. The intention with hardcoding the path here here was to make the tests robust to default values changing; since defaulting happens outside of these functions I only want those integration/UI tests to change if those defaults do.